### PR TITLE
Fix classpreloader bin tool

### PR DESCRIPTION
**Pull request by seyfer**

### CODE REVIEW

**Comment by seyfer** [comment]
Please, merge and tag 1.0.1 version or whatever for update.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 composer.lock
 phpunit.xml
 vendor
+.idea
```
Comment (GrahamCampbell) [inline .gitignore:4]: no

**Comment by GrahamCampbell** [comment]
tests failing?

**Comment by GrahamCampbell** [comment]
nvm

**Comment by GrahamCampbell** [comment]
thx

**Comment by GrahamCampbell** [comment]
I've tagged 1.0.1.

**Pull request merged**